### ISSUE TITLE
On creating new site get Server Error (500)

### ISSUE THREAD

**Issue by neilh10** [ODM2/ODM2DataSharingPortal#537]
On creating a new site, I get a Server Error (500)

![image](https://user-images.githubusercontent.com/2766619/146259811-15c695b3-b5d1-4bd8-8079-107e66cb311f.png)

however it is created.
Initially I created it with no notifications, but then went in and edited the site and change to Notifications, and saved and got the 500
I tried updating Site information with "Edit", and changed the "Notes" to slightly different text   
then Save edits 
I again get Server Error (500)

It appears to be repeatable.
I tried creating a new site https://monitormywatershed.org/sites/tu_rc_xx99/ and the same result - Server Error (500)

**Comment by neilh10** [comment]
On creating a sensor, adding the details, and pressing "Add New Sensor", the window never says its complete, 
![image](https://user-images.githubusercontent.com/2766619/146262016-ced87cad-13ab-4c6a-a22a-c006497798d3.png)

however if the CANCEL is pressed, and then browser refreshed, the sensor has been created.

![image](https://user-images.githubusercontent.com/2766619/146262174-07639f9d-3ed3-4e6c-a8a5-e66d21f7f93d.png)

**Comment by aufdenkampe** [comment]
@neilh10, thanks for this report. We'll look into it. 

We were doing some hot-fixes to the database yesterday, but those would have only resulted in momentary issues an it seems that your observations were repeatable. Thanks!

**Comment by ptomasula** [comment]
@neilh10, thanks for the testing and reporting this issue. This looks to have been caused by some old TSA related syncing code that was still running on the application. Because we're removed TSA, the end point for that code was not available and it produced the 500 response that you reported.

@aufdenkampe as a hot fix, I've commended out that code and will push an update to the application shorted. Longer term we'll go through the code base and do a proper refactoring and clean up. I'm sure this isn't the only piece of code in there. Looking forward to [milestone 19](https://github.com/ODM2/ODM2DataSharingPortal/milestone/19)
